test-utils/jest.setup.ts

```diff
@@ -1,7 +1,7 @@
 import { InternalDiscordManager } from './setup';
 import { server } from './mock-server';
 
-process.env.DISCORD_BOT_TOKEN = 'FAKE_BOT_TOKEN';
+process.env.DISCORD_TOKEN = 'FAKE_BOT_TOKEN';
 process.env.AIRTABLE_TOKEN = 'FAKE_AIRTABLE_TOKEN';
 process.env.AIRTABLE_BASE = 'FAKE_TABLE_NAME';
 process.env.GUILD_ID = 'DEV_DAO_ID';
```

Comment:
This is a bit awkward, because `./setup` is imported and evaluated before the environment variables are set, and the Discord Client is created as `export const client = ...` it means that the `token: process.env.DISCORD_TOKEN` line is evaluating to undefined.
So if you try to run the tests without a .env file locally it still doesn't pass.
I have tried moving all of the `process.env.X = Y` lines to the very top of this file and it works, but looks strange, and I have tried moving them into `test-utils/setup.ts` before the client is exported and that works too.

I think that the original change would have worked if `createFakeClient()` still needed to be executed before the Discord Client was instantiated, as the env vars would have been set.

This is pretty minor, because most people will have to create a `.env` file to do any manual testing or execution of the code anyway, but it would be nice to figure out the best way to get those values set before the client is created.

Reply:
What do you think to this?

1) Add in a new file `test.env` with the following contents
```bash
DISCORD_TOKEN=FAKE_BOT_TOKEN
AIRTABLE_TOKEN=FAKE_AIRTABLE_TOKEN
AIRTABLE_BASE=FAKE_TABLE_NAME
GUILD_ID=DEV_DAO_ID
CLIENT_ID=FAKE_CLIENT_ID
```
2) Add this to the very top of `jest.setup.ts`
```javascript
import dotenv from 'dotenv';
dotenv.config({ path: `${__dirname}/test.env` });
```

`test.env` is not caught by the `.gitignore` so will be included on everyone's machines and this allows anyone to just clone the repo and run the tests (after an `npm install`)

Reply:
Thanks for looking into it in-depth @AidenMontgomery - you're absolutely right. I'll address it now.

Reply:
@AidenMontgomery Jest also allows us to specify files to run before others, so I went that route just because it would mean one less dependency, but I'm open to being challenged in case you see more benefits from your original suggestion.

Reply:
I'll go ahead and merge this now just so I have this out of my mind, but happy to circle back to this in case you want to.